docs/source/en/generation_strategies.md

```diff
@@ -55,7 +55,7 @@ When you load a model explicitly, you can inspect the generation configuration t
 >>> from transformers import AutoModelForCausalLM
 
 >>> model = AutoModelForCausalLM.from_pretrained("distilgpt2")
->>> model.generation_config
+>>> model.generation_config  # doctest: +IGNORE_RESULT
```

Comment:
Skip: `transformers_version` is not a loaded field, it contains a reference to the current version -> perhaps it should not be part of the `__repr__`, but I'd like to defer that to a different PR